modules/4337/certora/requirements.txt

```diff
@@ -0,0 +1 @@
+certora-cli==7.3.0
```

Comment:
can we switch to latest https://pypi.org/project/certora-cli/ ? would be a good thing to do together with updating the scripts

Reply:
As mentioned in today's sync, I will create a new issue for the upgrade as this requires some changes to the specs which I want to discuss with Certora first.